packages/components/src/List/ListComposition/TextFilter/TextFilter.component.test.js

```diff
@@ -46,64 +46,68 @@ describe('TextFilter', () => {
 			textFilter: '',
 			setTextFilter: jest.fn(),
 		};
+		const event = { target: { value: 'my-filter-value' } };
 
 		// when
 		let wrapper;
 		act(() => {
 			wrapper = mount(
 				<ListContext.Provider value={context}>
-					<TextFilter id="myTextFilter" initialDocked />
+					<TextFilter id="myTextFilter" initialDocked debounceTimeout={0} />
 				</ListContext.Provider>,
 			);
 
-			wrapper
-				.find('FilterBar')
-				.at(0)
-				.prop('onToggle')();
-			wrapper
-				.find('FilterBar')
-				.at(0)
-				.prop('onFilter')({}, 'my-filter-value');
+			wrapper.find('button#myTextFilter').simulate('click');
```

Comment:
```suggestion
			wrapper.find('#myTextFilter').simulate('click');
```

Reply:
no that won't work. The enzyme tree is a component tree. So the id is passed to a component, that has the id, and pass it to the button component. So the find will return an array with multiple components that has the id as props.